src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -151,7 +173,8 @@ public boolean hasLog10PError() {
      */
     public double getPhredScaledQual() { return (getLog10PError() * -10) + 0.0; }
 
-    public void setLog10PError(double log10PError) {
+    /** set the Phred scaled quality score */
```

Comment:
Is this value Phred-scaled at this point?  It will become the QUAL (in the writer?), but I thought it's just stored as log10prob.